doc/nrf/glossary.rst

```diff
@@ -697,6 +697,7 @@ Glossary
 
    Toolchain
       A set of development tools.
+      The |NCS| toolchain is based on the GNU ARM Embedded Toolchain supported in the Zephyr SDK plus a set of tools specific to the |NCS|.
```

Comment:
```suggestion
      The |NCS| toolchain is based on the GNU ARM Embedded Toolchain supported in the Zephyr SDK plus a set of tools specific to the |NCS|.
```

Is this a correct statement? @gmarull 
Is this a correct statement? @gmarull

Reply:
No, this is wrong. `GNU ARM Embedded Toolchain` is another toolchain.